index.html

```diff
@@ -197,6 +197,51 @@ <h3 class="lang-h3">
         </ul>
       </div>
     </div>
+    <section class="contact-cont">
+      <div class="contact-margin">
+        <div class="contact-title">
+          <div class="contact-h1">
+            Get started
+          </div>
+          <div class="about-p p-height">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.
+          </div>
```

Comment:
- [ OPTIONAL ] I recommend to put here h2 or h3 tag instead of a div and p tag for the text that's very important to make your page folllow best HTML prcatice and to make your HTML semcantic 💪🏻💯🔥